src/components/pages/Offers/Offer/OfferDetails/OfferThumbnail/OfferThumbnail.scss

```
@@ -62,3 +79,8 @@
     opacity: 1;
   }
 }
+
+.of-error-upload-image {
```

```suggestion
.of-error {
```
of indique déjà "offer